app.js

```diff
@@ -29,7 +29,7 @@ winston.add(
 const models = require('./database/models');
 
 // Sync Database
-models.sequelize.sync().then(function() {
+models.sequelize.sync({force: true}).then(function() {
```

Comment:
You will have to remove this right ? It will cause problem when put in production.